rest/oidc_api.go

```diff
@@ -228,7 +228,7 @@ func (h *handler) getOIDCProvider(providerName string) (*auth.OIDCProvider, erro
 
 // Builds the OIDC callback based on the current request and database. Used during OIDC Client lazy initialization.  Needs to pass
 // in dbName, as it's not necessarily initialized on the request yet.
```

Comment:
Update comment to describe new parameters.

```suggestion
// Builds the OIDC callback based on the current request. Used during OIDC Client lazy initialization.
// Need to pass providerName and isDefault for the requested provider to determine whether we need to append it to the callback URL or not.
```

Reply:
This change has been comiitted